src/styles/Banner.css

```diff
@@ -112,6 +135,10 @@
     flex-direction: column;
     gap: 0;
   }
+
+  .modal_signup_btn {
```

Comment:
I think this code for `modal_signup_btn`  has already been written in line number `88`, so maybe we can add the style there ?